pkg/cluster/delete.go

```diff
@@ -398,8 +398,14 @@ func (m *manager) deleteResourcesAndResourceGroup(ctx context.Context) error {
 }
 
 func (m *manager) Delete(ctx context.Context) error {
+	m.log.Printf("running ensureResourceGroup")
+	err := m.ensureResourceGroup(ctx) // re-create RP RBAC if needed/missing
+	if err != nil {
+		return err
+	}
```

Comment:
As @gvanderpotte said, I think we should just log out this error.  It's best-effort and could have consequences in environments where the resource group already exists.

Reply:
@bennerv @gvanderpotte changed to log the error message from returning it.